parallel-install/pkg/prerequisites/prerequisites.go

```diff
@@ -34,17 +36,25 @@ func InstallPrerequisites(ctx context.Context, kubeClient kubernetes.Interface,
 	go func() {
 		defer close(statusChan)
 
-		config.Log("Creating kyma-installer namespace")
-		_, err := kubeClient.CoreV1().Namespaces().Create(context.Background(), &v1.Namespace{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:   "kyma-installer",
-				Labels: map[string]string{"istio-injection": "disabled", "kyma-project.io/installation": ""},
-			},
-		}, metav1.CreateOptions{})
+		config.Log("Deploying kyma-installer namespace")
 
-		if err != nil {
-			statusChan <- fmt.Errorf("Unable to create kyma-installer namespace. Error: %v", err)
+		_, err := kubeClient.CoreV1().Namespaces().Get(context.Background(), "kyma-installer", metav1.GetOptions{})
+
+		if errors.IsNotFound(err) {
+			nsErr := createNamespace(kubeClient)
+			if nsErr != nil {
+				statusChan <- fmt.Errorf("Unable to create kyma-installer namespace. Error: %v", nsErr)
+				return
+			}
+		} else if err != nil {
```

Comment:
I think it's more idiomatic to have `if err != nil` as the first/top-level "if" statement. And do more specific checks (like: `if errors.IsNotFound(err)`) as a second-level checks.